From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Fri, 22 Mar 2019 20:00:49 +0100
Subject: [PATCH] {MAYBE UPS} arch/arm/mm: remove unused variables

arm-gcc reports:

    arch/arm/mm/init.c: In function 'mem_init':
    arch/arm/mm/init.c:471:13: error: unused variable 'itcm_end'
    [-Werror=unused-variable]
      extern u32 itcm_end;
                 ^~~~~~~~
    arch/arm/mm/init.c:470:13: error: unused variable 'dtcm_end'
    [-Werror=unused-variable]
      extern u32 dtcm_end;
                 ^~~~~~~~
---
 arch/arm/mm/init.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/arm/mm/init.c b/arch/arm/mm/init.c
index 749a5a6f6143..b9b7513a66fc 100644
--- a/arch/arm/mm/init.c
+++ b/arch/arm/mm/init.c
@@ -447,12 +447,6 @@ static void __init free_highpages(void)
  */
 void __init mem_init(void)
 {
-#ifdef CONFIG_HAVE_TCM
-	/* These pointers are filled in on TCM detection */
-	extern u32 dtcm_end;
-	extern u32 itcm_end;
-#endif
-
 	set_max_mapnr(pfn_to_page(max_pfn) - mem_map);
 
 	/* this will put all unused low memory onto the freelists */
-- 
